src/m365/aad/commands/o365group/o365group-recyclebinitem-list.ts

```diff
@@ -0,0 +1,80 @@
+import * as chalk from 'chalk';
+import { Logger } from '../../../../cli';
+import {
+  CommandOption
+} from '../../../../Command';
+import GlobalOptions from '../../../../GlobalOptions';
+import { GraphItemsListCommand } from '../../../base/GraphItemsListCommand';
+import commands from '../../commands';
+import { Group } from './Group';
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  displayName?: string;
+  mailNickname?: string;
+}
+
+class AadO365GroupRecycleBinItemCommand extends GraphItemsListCommand<Group> {
+  public get name(): string {
+    return `${commands.O365GROUP_RECYCLEBINITEM_LIST}`;
+  }
+
+  public get description(): string {
+    return 'Lists Microsoft 365 Groups deleted in the current tenant';
+  }
+
+  public getTelemetryProperties(args: CommandArgs): any {
+    const telemetryProps: any = super.getTelemetryProperties(args);
+    telemetryProps.displayName = typeof args.options.displayName !== 'undefined';
+    telemetryProps.mailNickname = typeof args.options.mailNickname !== 'undefined';
+    return telemetryProps;
+  }
+
+  public defaultProperties(): string[] | undefined {
+    return ['id', 'displayName', 'mailNickname'];
+  }
+
+  public commandAction(logger: Logger, args: CommandArgs, cb: () => void): void {
+    const filter: string = `?$filter=groupTypes/any(c:c+eq+'Unified')`;
+    const displayNameFilter: string = args.options.displayName ? ` and startswith(DisplayName,'${encodeURIComponent(args.options.displayName).replace(/'/g, `''`)}')` : '';
+    const mailNicknameFilter: string = args.options.mailNickname ? ` and startswith(MailNickname,'${encodeURIComponent(args.options.mailNickname).replace(/'/g, `''`)}')` : '';
+    const topCount: string = '&$top=100';
+
+    let endpoint: string = `${this.resource}/v1.0/directory/deletedItems/Microsoft.Graph.Group${filter}${displayNameFilter}${mailNicknameFilter}${topCount}`;
+    
+
+    this
+      .getAllItems(endpoint, logger, true)
+      .then((): void => {
+        
+        logger.log(this.items);
+
+        if (this.verbose) {
+          logger.logToStderr(chalk.green('DONE'));
+        }
+
+        cb();
+      }, (err: any): void => this.handleRejectedODataJsonPromise(err, logger, cb));
+  }
+
+  public options(): CommandOption[] {
+    const options: CommandOption[] = [
+      {
+        option: '-d, --displayName [displayName]',
+        description: 'Lists groups with displayName starting with the specified value'
+      },
+      {
+        option: '-m, --mailNickname [displayName]',
+        description: 'Lists groups with mailNickname starting with the specified value'
```

Comment:
should be [mailNickname]. @waldekmastykarz not sure why we do not get 100% coverage here.

Do you have any clues?
![image](https://user-images.githubusercontent.com/18241305/103453241-42a9c000-4cd8-11eb-9a7e-25ccc07680ef.png)

Reply:
Typically, we include a test case for options. See https://github.com/pnp/cli-microsoft365/blob/6cbfea996d76edf189f6819778709eece1145dc3/src/m365/aad/commands/approleassignment/approleassignment-add.spec.ts#L358 for example

Reply:
just checked that now is getting 100% covered